cmds/commitlint.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const join = require('path').join
+const validate = require('validate-commit-msg')
```

Comment:
It seems this module has been deprecated in favor of https://github.com/marionebl/commitlint (module read-only Github repository: https://github.com/marionebl/commitlint)